packages/office-ui-fabric-react/src/components/Modal/examples/Modal.Basic.Example.scss

```diff
@@ -0,0 +1,29 @@
+@import '../../../common/common';
+
+:global {
+  .ms-modalMainOverride {
+    height: 80vh;
+    width: 80vw;
+    display: flex;
+    flex-flow: column nowrap;
+    align-items: stretch;
+
+    .ms-modalExample-header {
+      flex: 1 1 auto;
+      background: $ms-color-themePrimary;
+      color: #fff;
+      display: flex;
+      align-items: center;
+      @include ms-font-xxl;
+      @include ms-fontWeight-semibold;
+      padding-left: 5px;
+      min-height: 40px;
+    }
+
+    .ms-modalExample-body {
+      flex: 4 4 auto;
+      padding: 5px;
```

Comment:
If we're going to take a moment to style it, we should probably go with more than 5px padding. The dialog uses upwards of 28px.